src/roswire/ros1/format.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
```

Comment:
Note that this file and ROS2FormatDatabase will be changed to use ROS-version specific MsgFromat and ActionFormat in a subsequent PR.